k9mail-library/src/test/java/com/fsck/k9/mail/internet/MimeMessageParseTest.java

```diff
@@ -272,4 +273,60 @@ private static void checkLeafParts(MimeMessage msg, String... expectedParts) thr
         }
         assertEquals(Arrays.asList(expectedParts), actual);
     }
+
+    @Test
+    public void isXOriginalTo() throws Exception {
```

Comment:
This name doesn't describe what is being tested. Suggestion: `getRecipients_withXOriginalTo()`.